scheduler/src/cook/mesos/api.clj

```diff
@@ -108,9 +108,14 @@
 (def ZeroInt
   (s/both s/Int (s/pred zero? 'zero?)))
 
-(def PosNum
+(s/defschema PosNum
```

Comment:
What effect does `s/defschema` have?

Reply:
I think it attaches some metadata that's checked when generating the Swagger HTML page. We should probably change all of our schema definitions to use this macro.